### DISCUSSION TITLE
Rethink & Android Accessibility

### DISCUSSION THREAD

**Discussion by simple-basil** [celzero/rethink-app#1780] [General]
Hi there,

<br/>
I've noticed that Rethink app has capability to use Android Accessibility service <br><br/>

![Screenshot_1](https://github.com/user-attachments/assets/8e4ece05-5a17-4963-9ef2-8f0ab2b9a99a)

So, 

- Is it good or bad idea to allow Rethink using Accessibility service? cause Accessibility service is known for dramatical decreasing device perfomance and responsiveness while using certain apps... even on flaghip devices.

- Which features is Rethink app providing in conjunction with Accessibility service?

<br/>
Thank you.

**Comment by ignoramous** [comment]
> Is it good or bad idea to allow Rethink using Accessibility service?

Not if you don't need it. And yes, *Accessibility Service* (the limited parts that Rethink uses) may prove to be an extra drain on power use (but nothing much else).

> Which features is Rethink app providing in conjunction with Accessibility service?

*Configure* -> *Firewall* -> *Universal firewall rules* -> *Block any app not in use*.
